fhir-server/liberty-config/config/default/fhir-server-config-db2.json

```diff
@@ -100,25 +100,18 @@
             "jdbc": {
                 "__comment": "Configuration properties for the JDBC persistence implementation",
                 "bootstrapDb": true,
-                "dataSourceJndiName": "jdbc/fhirProxyDataSource",
+                "enableProxyDatasource": false,
+                "bootstrapDataSourceBase": "jdbc/bootstrap",
                 "enableCodeSystemsCache": true,
                 "enableParameterNamesCache": true,
                 "enableResourceTypesCache": true
             },
             "datasources": {
                 "default": {
                     "type": "db2",
+                    "currentSchema": "fhirdata",
                     "hints" : {
                       "search.reopt": "ONCE"
```

Comment:
Doesn't feel so much like deprecating, rather this is a breaking change.

Reply:
If you decide to use the new configuration, yes. The existing configuration will work the same as it does today without change - it will just be using a deprecated class under the covers.

Reply:
Note, I've reverted the Db2 example to use the old style configuration so that we can continue to exercise this code as part of SIT.